templates/docs/upgrade-guide-v3.md

```diff
@@ -20,11 +28,27 @@ The `$breakpoint-medium` variable has been removed from Vanilla. All media queri
 
 The default value of `$breakpoint-navigation-threshold` was previously set to `$breakpoint-medium` and is now `$breakpoint-small`. This value should be overridden in project code to adjust the threshold at which navigation switches to dropdown, based on the number of navigation items.
 
+In places where the `u-hide--small` class is currently used, `u-hide--medium` may need to be added. This is to replicate current behaviour of `u-hide--small`, ensuring the element is hidden on small and medium screens.
+
 ## Buttons
 
-The neutral button style provided by `p-button--neutral` was exactly the same as the default `p-button` styling, so it has been removed. Please use `p-button` instead.
+### Neutral buttons have been removed
+
+The neutral button style provided by `p-button--neutral` was exactly the same as the default `p-button` styling, so it has been removed. Please use `p-button` or class-less `<button>` element instead.
+
+### Full width mobile buttons have been removed
+
+In Vanilla 2.0 buttons on smallest screens (below `$breakpoint-x-small`) have been extended to full 100% width. This was causing issues on some layouts and with some utilities, so is no longer part of Vanilla 3.0. If your project has any custom code that was needed because of full width buttons you may need to update or remove it.
```

Comment:
```suggestion
In Vanilla 2.0 buttons on smallest screens (below `$breakpoint-x-small`) were extended to full 100% width. This was causing issues on some layouts and with some utilities, so is no longer part of Vanilla 3.0. If your project has any custom code that was needed because of full width buttons you may need to update or remove it.
```